### ISSUE TITLE
Incomplete support for FRAM on 4801

### ISSUE THREAD

**Issue by terrillmoore** [mcci-catena/Catena-Arduino-Platform#149]
This is not good. Catena-Arduino-Platform doesn't know how to turn on power on FRAM on the 4801 (or other devices that have FRAM power). This needs to be fixed, and examples must be updated to remove manual FRAM control. I'm really dismayed that this hasn't been bug-reported before this, as it's a major defect in design, and easily remedied.

**PRINCIPLE**
Sketches need to be portable between boards.

So if you find yourself having to do something for a given board that doesn't have to be done on a different board, **do not just do it**. File a bug report and serve as a test case for the new abstraction.

**Comment by dhineshkumarmcci** [comment]
Yes, with the test results, we have to add the FRAM, RS485 Transceiver and Booster Enable control in the Catena-Arduino-Platform.

In future, will make sure to file a bug, if any configuration is particular to a specific board.
